### ISSUE TITLE
alt

### ISSUE THREAD

**Issue by OnsNader** [FACG5/NextInTeam#21]
https://github.com/FACG5/NextInTeam/blob/9fae54b1cbb57d9ab755089906ebf6931b0aa30d/index.html#L13
No alt impacts accessibility.

**Comment by OnsNader** [comment]
Second header should be an h2 not h3. It's too small.  But all in all, your website looks unique.
